cpp/frontend_match_test.cc

```diff
@@ -8,14 +8,15 @@ namespace Id = id_kind_asdl::Id;
 TEST match_test() {
   match::SimpleLexer* lex = match::BraceRangeLexer(StrFromC("{-1..22}"));
 
-  while (true) {
-    auto t = lex->Next();
-    int id = t.at0();
+  List<Tuple2<Id_t, Str*>*>* toks = lex->Tokens();
```

Comment:
Did this test have to change?

It seems like we're not testing `Next()` anymore?

Can we have both tests for coverage?   i.e. the old one and the new one


I may have forgotten how this code works ...

Reply:
Ah `Tokens()` calls `Next()` here: https://github.com/oilshell/oil/blob/master/cpp/frontend_match.cc#L44-L46

Reply:
This change wasn't for any fundamental reason, though. Just a cheap way to bump up coverage since the old loop from the test and `Tokens()` are nearly identical